README.md

```diff
@@ -9,6 +9,7 @@ In order to use this action, you need to:
 1. [Register new GitHub App](https://docs.github.com/apps/creating-github-apps/setting-up-a-github-app/creating-a-github-app)
 2. [Store the App's ID in your repository environment variables](https://docs.github.com/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows) (example: `APP_ID`)
 3. [Store the App's private key in your repository secrets](https://docs.github.com/actions/security-guides/encrypted-secrets?tool=webui#creating-encrypted-secrets-for-a-repository) (example: `PRIVATE_KEY`)
+4. We recommend that `owner` be set to `env.GITHUB_REPOSITORY_OWNER` (See [environment variables](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables))
 
 ### Minimal usage
```

Comment:
We can rename it to make clear what it dose

```suggestion
### Create token for current repository
```

I would then add more usage examples

1. Create token for all repositories in current owner's installation
2. Create a token for selected repositories in current owner's installation
3. Create token for all repositories in another owner's installation

Note that an owner can be both an organization or a user

Reply:
Ah gotcha, that makes sense. Done!